src/internal/packager/images/push.go

```diff
@@ -113,43 +88,18 @@ func Push(ctx context.Context, cfg PushConfig) error {
 				return err
 			}
 
-			message.Debugf("push %s -> %s)", refInfo.Reference, offlineName)
-
 			if err = pushImage(img, offlineName); err != nil {
 				return err
 			}
 
 			pushed = append(pushed, refInfo)
-			totalSize -= size
 		}
-		progress.Successf("Pushed %d images", len(cfg.ImageList))
 		return nil
 	}, retry.Context(ctx), retry.Attempts(uint(cfg.Retries)), retry.Delay(500*time.Millisecond))
 	if err != nil {
 		return err
 	}
+	spinner.Success()
```

Comment:
Using `spinner.Sucess()` gives the spinner a check mark. I put examples in the PR description of what it looks like after the images for a component are complete

Reply:
Changed to move back to `message.Infof`. I was between that and the spinner. The spinner is more consistent with out current UI. `message.Info` is more verbose, but that may help in debugging issues. Either way this will likely change in the future as we do the log refactor.